content/en/integrations/syslog_ng.md

```diff
@@ -69,8 +69,14 @@ Configure Syslog-ng to gather logs from your host, containers, & services.
     ...
 
     # For Datadog platform:
-    template DatadogFormat { template("YOURAPIKEY <${PRI}>1 ${ISODATE} ${HOST:--} ${PROGRAM:--} ${PID:--} ${MSGID:--} ${SDATA:--} $MSG\n"); };
-    destination d_datadog { tcp("intake.logs.datadoghq.com" port(10514) template(DatadogFormat)); };
+    destination d_datadog {
+      http(
+          url("https://http-intake.logs.datadoghq.com/api/v2/logs?ddsource=<SOURCE>&ddtags=<TAG_1:VALUE_1,TAG_2:VALUE_2>")
```

Comment:
From the looks of it this is an update for the us3 region, so it should point to the us3 intake:
`https://http-intake.logs.us3.datadoghq.com/api/v2/logs`